### PR TITLE
Added image labels to satisfy EC

### PR DESCRIPTION
**Pull request by syedriko**
Added image labels to satisfy EC

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -25,3 +25,17 @@ RUN set -e && for OCP_VERSION in $(ls -1 ocp-product-docs-plaintext); do \
 FROM scratch
 COPY --from=lightspeed-rag-builder /workdir/vector_db/ocp_product_docs /rag/vector_db/ocp_product_docs
 COPY --from=lightspeed-rag-builder /workdir/embeddings_model /rag/embeddings_model
+
+# Labels for enterprise contract
+LABEL com.redhat.component=openshift-lightspeed
+LABEL description="Red Hat OpenShift Lightspeed - AI assitant for managing OpenShift clusters."
+LABEL distribution-scope=public
```
Comment (xrajesh) [inline Containerfile:32]: Do we want this public , we dont release Rag content right ?

File: Containerfile [hunk 2]
```diff
@@ -25,3 +25,17 @@ RUN set -e && for OCP_VERSION in $(ls -1 ocp-product-docs-plaintext); do \
...
+LABEL io.k8s.description="Red Hat OpenShift Lightspeed - AI assitant for managing OpenShift clusters."
+LABEL io.k8s.display-name="Openshift Lightspeed"
+LABEL io.openshift.tags="openshift,lightspeed,ai,assistant"
+LABEL name=openshift-lightspeed
```
Comment (xrajesh) [inline Containerfile:36]: should this be `lightspeed-rag-content`

File: Containerfile (hunk 1 above)
Comment (syedriko) [reply Containerfile:32]: by that logic this image shouldn't be checked against EC because we don't ship it.

File: Containerfile [hunk 3]
```diff
@@ -25,3 +25,17 @@ RUN set -e && for OCP_VERSION in $(ls -1 ocp-product-docs-plaintext); do \
 FROM scratch
 COPY --from=lightspeed-rag-builder /workdir/vector_db/ocp_product_docs /rag/vector_db/ocp_product_docs
 COPY --from=lightspeed-rag-builder /workdir/embeddings_model /rag/embeddings_model
+
+# Labels for enterprise contract
+LABEL com.redhat.component=openshift-lightspeed
```
Comment (tisnik) [inline Containerfile:30]: Is it ok to have the same component as the operator?

In service we have:
com.redhat.component=openshift-lightspeed-service

File: Containerfile (hunk 2 above)
Comment (tisnik) [reply Containerfile:36]: yup, should be different

File: Containerfile [hunk 4]
```diff
@@ -25,3 +25,17 @@ RUN set -e && for OCP_VERSION in $(ls -1 ocp-product-docs-plaintext); do \
...
+LABEL distribution-scope=public
+LABEL io.k8s.description="Red Hat OpenShift Lightspeed - AI assitant for managing OpenShift clusters."
+LABEL io.k8s.display-name="Openshift Lightspeed"
+LABEL io.openshift.tags="openshift,lightspeed,ai,assistant"
+LABEL name=openshift-lightspeed
+LABEL release=0.0.1
+LABEL url="https://github.com/openshift/lightspeed-operator"
+LABEL vendor="Red Hat"
+LABEL version=0.0.1
+LABEL summary="Red Hat OpenShift Lightspeed"
```
Comment (tisnik) [inline Containerfile:41]: should be more precise IMHO (see what's in operator and service containerfiles)

**Comment by syedriko** [comment]
@tisnik @xrajesh PTAL

File: Containerfile (hunk 1 above)
Comment (syedriko) [reply Containerfile:32]: Found possible values of distribution-scope in https://github.com/projectatomic/ContainerApplicationGenericLabels, made it private.

**Review by tisnik — COMMENTED** [review]
it looks correct

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-e2e-tests-rosa / lightspeed-rag-content "

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-rag-content"

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-e2e-tests-rosa / lightspeed-rag-content"

**Pull request merged**